docs/tutorials/helloworldparttwo.md

```diff
@@ -209,6 +207,9 @@ namespace HelloWorld
     }
 }
 ```
+16. Select the Player Prefab
```

Comment:
Period, also if a selectable option on screen, bolding the name of it **Player Prefab**?
```suggestion
16. Select the Player Prefab.
```

Reply:
yep bold and i will learn to add sodding periods